README.md

```diff
@@ -253,7 +253,8 @@ Brief explanation of the options in [maker.py](./TreeMaker/python/maker.py)
 * `saveMinimalGenParticles`: save only the hard scatter gen particles coming from top decays, boson decays, semi-visible jets, or SUSY particles (default=True)
 * `saveGenTops`: save the 4-vectors of the generated tops and the TTbar reweighting scale factor (default=False)
 * `doMT2`: switch to enable the storage of the MT2 variable (default=False)
-* `nestedVectors`: switch to change from saving vector<\vector<T>> to saving vector<T> values and vector<\int> offsets (default=True)
+* `nestedVectors`: switch to change from saving vector<\vector<T>> to saving vector<T> values and vector<\int> counts (default=True)
+* `nestedCounts`: if set to False, stores offsets rather than counts when using `nestedVectors=False` (default=True)
```

Comment:
I think you mean "when using `nestedVectors=True`"?

Reply:
so `nestedVectors=True` means not doing the flattening, `nestedVectors=false` means do the flattening. If doing the flattening, `nestedCounts=False` would use offsets rather than counts.

I would almost suggest changing both cl options to `unnestedVectors` and `unnestedCounts`...

Reply:
Okay, this is definitely getting confusing.

By itself, it makes sense if `nestedVectors=True` means store `vector<vector<T>>`. But it's very confusing if `nestedCounts` only applies when `nestedVectors` is disabled. I think we need to change `nestedCounts` to something clearer. Maybe `storeOffsets`, set to `False` by default, so it's clear that the default option is now storing counts?

We may also want to disable `nestedVectors` by default at this point.